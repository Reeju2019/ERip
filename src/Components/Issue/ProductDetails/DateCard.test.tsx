import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import DateCard from './DateCart'
// import jest from "jest"


const setActiveDate= jest.fn()

const item = ["a","b","c"]
test('date card', () => {
  render(<DateCard date={item} setActiveDate={setActiveDate}/>)
})
describe('App rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<DateCard date={item} setActiveDate={setActiveDate}/>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<DateCard date={item} setActiveDate={setActiveDate}/>)
    ReactDOM.unmountComponentAtNode(div)
  })
})
