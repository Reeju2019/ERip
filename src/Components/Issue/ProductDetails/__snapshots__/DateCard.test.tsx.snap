// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering specification SnapShot testing 1`] = `
<div
  className=""
  onClick={[Function]}
>
  <h6>
    a
  </h6>
  <h4>
    b
  </h4>
  <h6>
    c
  </h6>
</div>
`;
